### ISSUE TITLE
The navbar cache is at it again: CSRF super rude failure

### ISSUE THREAD

**Issue by niklasmohrin** [e-valuation/EvaP#1645]
TLDR: After #1616 we have a form in the navbar again, invalidating the evaluation at https://github.com/e-valuation/EvaP/issues/1208#issuecomment-398153367. Thus, we made a full cycle back to #862. It is exactly the same problem.

---

From the makers of _[CSRF problems on language change](https://github.com/e-valuation/EvaP/issues/862)_ and _[CSRF 2: Navbar cache refresh is broken](https://github.com/e-valuation/EvaP/issues/1208)_ and the creator of [the infamous language change spinner](https://github.com/e-valuation/EvaP/pull/1616): The all new _CSRF 3: CSRF super rude failure_.

The navbar cache is at it again. In this iteration of the renown franchise, doubt reigns upon the once-beloved django feature. How will the EvaP developer team respond this time? Will they fix the problem in exactly the same way they did 5 years ago? Or will they eliminate the navbar cache once and for all?

**Comment by He3lixxx** [comment]
@janno42 voted in favor of removing the navbar cache just as proposed in #1561. However, our result there seems to be that the navbar cache reduces page load time by ~100ms(?), which would make me favor keeping it (although we should probably add a test for changing the language).

**Comment by He3lixxx** [comment]
I think we've changed a bit of stuff on the navbar in the meantime.

Current benchmarks on my machine (I5-6200U):
9a9926d3, http://localhost:8000/rewards/, logged in as `evap`, freshly reloaded testdata, measuring request duration in firefox (request -> response received), in milliseconds:
With navbar cache: 44, 36, 43, 48, 36, 43, 38, 34, 35, 38. Average 39.5
Without navbar cache: 63, 71, 55, 57, 46, 72, 71, 50, 63, 52. Average 60.

So seems like we're talking about 20ms with our current test data. We cause 4 additional queries: group, group, responsible-status, semester.

I'll look into whether we can reduce that number, but overall, I think we should just remove the cache at this point.
